charts/gardener-extension-provider-alicloud/templates/deployment.yaml

```diff
@@ -34,6 +34,11 @@ spec:
         prometheus.io/port: "{{ .Values.metricsPort }}"
         {{- end }}
       labels:
+        networking.gardener.cloud/to-runtime-apiserver: allowed
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-private-networks: allowed
+        networking.resources.gardener.cloud/to-all-shoots-kube-apiserver-tcp-443: allowed
```

Comment:
I may not have complete understanding of whole solution. but how about gardener-extension-admission-alicloud running garden runtime cluster? is the new solution also applicable for it?

Reply:
Good question! The admission component in the garden cluster will be adapted separately in the future. It is out-of-scope for now. :)